Content.Shared/Storage/EntitySystems/SharedStorageSystem.cs

```diff
@@ -1522,6 +1532,27 @@ private bool ValidateInput(EntitySessionEventArgs args,
         return true;
     }
 
+    /// <summary>
+    /// Check if the entity interacting with the container is doing so with an empty hand.
+    /// </summary>
+    private bool ValidateEmptyHand(EntitySessionEventArgs args, [NotNullWhen(true)] out Hand? emptyHand)
```

Comment:
IMO a name like `TryGetEmptyHand` (to parallel `TryComp`) fits better with the `out var` semantics.
But at the same time I have next to no experience with `out var`s, so I dunno.